types/exaroton/index.d.ts

```diff
@@ -287,80 +157,117 @@ declare class File {
 
     setPath(path: string): void;
 
-    /**
-     * Apply data from the API Response
-     */
     applyData(object: object): File;
 
-    /**
-     * Set the server
-     *
-     * @param server
-     */
     setServer(server: Server): File;
 
-    /**
-     * Set the API client
-     */
     setClient(client: Client): File;
 
-    /**
-     * Get file information from the API
-     */
     getInfo(): Promise<File>;
 
-    /**
-     * Get the data/content of a file
-     *
-     * If you want to download the file to a local file use File.download() instead
```

Comment:
Some of these JSDoc comments seemed useful. Why remove them?

Reply:
Aren't they sort of duplicate if the source package already has JSDoc comments?

Reply:
Nope! TS Server will not read the source package because it only needs to read the types in order to provide intellisense.

Reply:
Oh, I didn't know that.
Then of course I'll add them back again. ^^